action.yml

```
@@ -0,0 +1,49 @@
+name: "github-action-auto-context"
+description: "Automaticaly open a pull request to update context.tf for Terraform projects"
+inputs:
+  branch-name:
+    description: "Name of branch to commit updated context.tf to"
+    required: false
+    default: "auto-update/context.tf"
+  token:
+    description: "Token for authenticating to GitHub API server. No special permissions needed."
+    required: true
+  bot-name:
+    decription: "Username to write commits under"
+    required: false
+    default: "cloudpossebot"
+runs:
+  using: "composite"
+  steps:
+    - name: "Checkout commit"
+      uses: actions/checkout@v2
+
+    - name: "Update context.tf, checkout new branch, and commit new context.tf, if needed"
+      id: update
+      uses: docker://cloudposse/build-harness:latest
+      with:
+        entrypoint: /github/workspace/context_update.sh
+      env:
+        BRANCH_NAME: ${{ inputs.branch-name }}
+        BOT_NAME: ${{ inputs.bot-name }}
+        HOST_REPO: ${{ github.repository }}
+
+    - name: Create Pull Request
+      uses: cloudposse/actions/github/create-pull-request@0.30.0
+      with:
+        token: ${{ inputs.token }}
+        committer: '${{ inputs.bot-name }} <11232728+${{ inputs.bot-name }}@users.noreply.github.com>'
+        author: '${{ inputs.bot-name }} <11232728+${{ inputs.bot-name }}@users.noreply.github.com>'
```

```suggestion
        committer: '${{ inputs.bot-name }} <11232728+${{ inputs.bot-name }}@users.noreply.github.com>'
        author: '${{ inputs.bot-name }} <11232728+${{ inputs.bot-name }}@users.noreply.github.com>'
```
11232728 is cloudposse specific. It won't work for others.

---

add `bot-email`